apps/baseline/serializers.py

```diff
@@ -1466,3 +1470,175 @@ def get_strategy_label(self, obj):
 
     def get_wealth_group_label(self, obj):
         return str(obj.wealth_group)
+
+
+class DictQuerySetField(rest_framework_fields.SerializerMethodField):
```

Comment:
I think this needs a class docstring explaining why it is required / what it does.